cextern/wcslib/C/wcshdr.c

```diff
@@ -1946,8 +1946,7 @@ void wcshdo_format(
   int emax = -999;
   int emin = +999;
   int precision = 0;
-  int i;
-  for (i = 0; i < nval; i++) {
+  for (int i = 0; i < nval; i++) {
```

Comment:
tl;dr -- Will this be patched upstream too? Otherwise the next bundle update will overwrite this.

Reply:
this is "unpatching" my patch, though I think it is just harmless to leave it as is for now.